lib/bounces.js

```diff
@@ -47,6 +47,19 @@ module.exports = function (log, error) {
         // But we can measure it!
         log.increment('account.email_bounced.already_verified')
       }
+      // Because we have no session token on this endpoint, metrics context
+      // metadata was stashed against a synthesized token for the benefit of
+      // the activity events. This fake request object allows the correct
+      // metadata to be gathered when we emit the events.
+      var fakeRequestObject = {
```

Comment:
I missed this PR before, otherwise I'd have commented on it sooner, sorry.

Anyway, does this work? Metrics context data is stashed per-event so, in order to `gather` it back out, it must have first been stashed using the appropriate event name. In other words, it will (or should, unless my understanding is broken) need a matching call to `metricsContext.stash(..., 'account.email_bounced', ...)` at some point.